packages/fxa-auth-server/lib/payments/paypal-client.ts

```diff
@@ -4,6 +4,7 @@
 import pRetry from 'p-retry';
 import querystring from 'querystring';
 import superagent from 'superagent';
+import { ChargeCustomerOptions } from './paypal';
```

Comment:
It doesn't seem like the right separation of concerns to have a lower level client pull in types from the higher level helper library for its own use. Ideally the lower level client could be yanked out of FxA entirely on its own for use as a stand-alone library (though obviously we aren't going to do that at this time).

Reply:
I agree. Will change.